scipy/stats/_morestats.py

```diff
@@ -12,12 +12,12 @@
 from scipy import optimize
 from scipy import special
 from scipy._lib._bunch import _make_tuple_bunch
-from scipy._lib._util import _rename_parameter
+from scipy._lib._util import (_rename_parameter, _contains_nan)
```

Comment:
For consistency, let's not add parentheses unless they're necessary due to line break.

Reply:
```suggestion
from scipy._lib._util import _rename_parameter, _contains_nan
```